app/src/main/java/io/homeassistant/companion/android/bluetooth/ble/IBeaconMonitor.kt

```diff
@@ -82,4 +91,14 @@ class IBeaconMonitor {
         sensorManager.updateBeaconMonitoringSensor(context)
         SensorReceiver.updateAllSensors(context)
     }
+
+    public fun setUUIDFilter(uuidFilter: List<String>, uuidFilterExclude: Boolean) {
```

Comment:
`public` isn't necessary

```suggestion
    fun setUUIDFilter(uuidFilter: List<String>, uuidFilterExclude: Boolean) {
```